lib/rubocop/ast/node/block_node.rb

```diff
@@ -32,6 +35,23 @@ def arguments
         end
       end
 
+      # Returns a collection of all descendants of this node that are
+      # argument type nodes. See `ArgsNode#argument_list` for details.
+      #
+      # @return [Array<Node>]
+      def argument_list
+        return [] unless arguments?
+
+        arguments.argument_list
+      end
+
+      # The argument names of this block.
+      #
+      # @return [Array<Symbol>]
+      def argument_names
```

Comment:
I'm not sure this method is needed. Let's remove it for now and if we keep using it we can create a shortcut

Reply:
Yeah I was hoping to squeak it in because I want it in my rubocop PR but that sounds reasonable 😅